client/app/lib/flux/teams/actions.coffee

```diff
@@ -347,6 +348,21 @@ fetchCurrentStateOfApiAccess = ->
   state = team.isApiEnabled is yes
   reactor.dispatch actions.SET_API_ACCESS_STATE, { state }
 
+fetchOtaToken = ->
```

Comment:
We talked the exact same thing and decided to make all the functions renamed to `load` prefix instead of `fetch` prefix. We will do it in another PR.